### ISSUE TITLE
 broke up with my gf of 4 years tonight. What do I do next?

### ISSUE THREAD

**Issue by stavratum** [HardGameSs/PSX#43]
I'm going to make this short, or else I know nobody will read it.

We're both sophomores in college, but we go to school on opposite sides of the state so it's been a long distance relationship for a while. My girlfriend and I had a long phone conversation that ended up with her essentially saying that she felt both intimidated and guilty that I was so devoted to our relationship working out, so we broke up. This conversation had come up before this past January, and I knew she was sincere. She's had doubts about us for a while now, and I feel like I owe it to her to do this.

Problem is, I hate it. And I'm lost as to what I do next. Any suggestions? Any good music or books or ideas or something to keep my brain occupied?

TL;DR - I broke up with a woman I love dearly and am now lost. Help.
